frontend/src/qri/searchResult.ts

```diff
@@ -0,0 +1,12 @@
+import { Dataset } from './dataset'
+
+export interface SearchResult extends Dataset {
+  follow_stats: {
+    follow_count: number
+    follow_status: number
+  }
+  issue_stats: {
+    open_issues: number
+    closed_issues: number
+  }
```

Comment:
@Arqu these fields are using snake case when they should be using camel case, we should get these converted to lowerCamelCase at some point.

@chriswhong to compensate for this API mismatch, switch these fields in lowerCamelCase & add a constructor function to this file  that is aware of the conversion:


```tsx
import { NewDataset } from './Dataset'


export function NewSearchResult(d: Record<string,any>): SearchResult {
  const ds = NewDataset(d)
  const followStats, issueStats

  if (d.follow_stats) {
    followStats = {
      followCount: d.follow_stats.follow_count,
      followStatus: d.follow_stats.follow_status
    }
  }

  if (d.issue_stats) {
    issueStats = {
      openIssues: d.issue_stats.open_issues,
      closedIssues: d.issue_stats.closed_issues
    }
  }

  return {
    ...ds,
    followStats: d.followStats || followStats,
    issueStats: d.issueStats || issueStats
  }
}
```

Use this constructor when mapping a response back from the server